src/platform/silabs/multi-ota/OTAFactoryDataProcessor.cpp

```diff
@@ -66,9 +66,11 @@ CHIP_ERROR OTAFactoryDataProcessor::ApplyAction()
     }
     else
     {
-        ChipLogProgress(SoftwareUpdate, "Factory data update finished.");
+        error = Provision::Manager::GetInstance().GetStorage().Commit();
```

Comment:
For EFR, no need to commit , please add the Macro for SiWG917.

Reply:
This will be needed for EFR devices as well if factory data multi-OTA on EFR32 is using the flash storage. Right now  factory data multi-OTA on EFR32 is using NVM.